City-of-Helsinki/drupal-hdbt
# UHF-X accordion state fix & accessibility fix

# [UHF-0000](https://helsinkisolutionoffice.atlassian.net/browse/UHF-0000)

## What was done

- Fixed problem which prevented accordion from working on multiple pages
- Added type attribute to button-html element due to accessibility guidelines

## How to install

* Make sure your instance is up and running on latest dev branch.
  * `git pull origin dev`
  * `make fresh`
* Update the HDBT theme
  * `composer require drupal/hdbt:dev-UHF-X_accordion_state_fix`
* Run `make drush-cr`

## How to test
- Open and close accordion items on multiple pages
- Reload the pages after opening and closing accordions
  - Accordion items should open and close on page load  
  - No javascript errors on inspector while opening and closing the accordion items after page reload
- Accordion toggle & close buttons have `type="button"` -attribute
